README.md

```diff
@@ -1 +1,93 @@
-# private-events
\ No newline at end of file
+# README
+
+This README would normally document whatever steps are necessary to get the
+application up and running.
+
+Things you may want to cover:
+
+* Ruby version![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Private_events
+
+In this private_event project we created a simple website where user must first login before creating events. Users also can exchange invitations only for their own events.
+
+## Built With
+
+- Ruby on Rails
+- Active Record
+- Bootstrap 4
+- Ruby
+
+## Getting Started
+
+To get a local copy of the repository please run the following commands on your terminal:
+
+```
+$ cd <folder>
+```
+
+```
+$ git clone git@github.com:verissimohenry/Private-Events.git 
+```
+
+```
+$ cd private_events
+```
+> Install webpack
+```
+$ rails webpacker:install 
+```
+
+> Migrate the database
+
+```
+$ rails db:migrate
+```
+$ bundle install
+```
```

Comment:
OPTIONAL

- [ ] Group all commands so that the instructions are clear and concise.